pynxtools/dataconverter/convert.py

```diff
@@ -34,11 +34,7 @@
 from pynxtools.dataconverter.template import Template
 from pynxtools.nexus import nexus
 
-
-logger = logging.getLogger(__name__)  # pylint: disable=C0103
-UNDOCUMENTED = 9
-logger.setLevel(logging.INFO)
-logger.addHandler(logging.StreamHandler(sys.stdout))
+from .logger import logger
```

Comment:
We should use absolute imports like we do in the rest of the project. The only upside is we can move this file anywhere and it will still work with an absolute import. Also it looks cleaner because we already do it everywhere else.